src/Workspaces/Core/Portable/Workspace/Solution/Project.cs

```diff
@@ -147,7 +147,12 @@ internal Project(Solution solution, ProjectState projectState)
     /// <summary>
     /// The options used by analyzers for this project.
     /// </summary>
-    public AnalyzerOptions AnalyzerOptions => State.AnalyzerOptions;
+    public AnalyzerOptions AnalyzerOptions => State.ProjectAnalyzerOptions;
+
+    /// <summary>
+    /// The options used by analyzers for this project.
+    /// </summary>
+    public AnalyzerOptions HostAnalyzerOptions => State.HostAnalyzerOptions;
```

Comment:
AnalyzerOptionsWithFallback?

Reply:
➡️ I find `Host` to be more meaningful than `WithFallback` in all cases except for the actual fallback implementation.